src/main/kotlin/no/nav/familie/ef/sak/opplysninger/personopplysninger/mapper/AdresseMapper.kt

```diff
@@ -65,13 +65,16 @@ class AdresseMapper(private val kodeverkService: KodeverkService) {
 
     private fun tilFormatertAdresse(bostedsadresse: Bostedsadresse, gjeldendeDato: LocalDate): String? {
         val (_, _, _, coAdressenavn, utenlandskAdresse, vegadresse, ukjentBosted, matrikkeladresse, _) = bostedsadresse
+        val coAdresse = coAdresse(coAdressenavn)
         val formattertAdresse: String? = when {
             vegadresse != null -> tilFormatertAdresse(vegadresse, gjeldendeDato)
             matrikkeladresse != null -> tilFormatertAdresse(matrikkeladresse, gjeldendeDato)
             utenlandskAdresse != null -> tilFormatertAdresse(utenlandskAdresse, gjeldendeDato)
-            else -> ukjentBosted?.bostedskommune
+            ukjentBosted != null -> "Ukjent bosted - ${ukjentBosted.bostedskommune}"
```

Comment:
Om man nu har bostedskommune da 😬

Reply:
Du tenker det ser dust ut hvis man mangler bostedskommune? At det blir seende ut slik; "Ukjent bosted -" for saksbehandler?